WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3932,7 +3932,12 @@ public void onMediaModelsCreatedFromOptimizedUris(@NonNull Map<Uri, ? extends Me
     }
 
     @Override public void showVideoDurationLimitWarning(@NonNull String fileName) {
-        ToastUtils.showToast(this, R.string.error_media_video_duration_exceeds_limit, ToastUtils.Duration.LONG);
+        String message = getString(R.string.error_media_video_duration_exceeds_limit);
+        WPSnackbar.make(
+                findViewById(R.id.editor_activity),
+                message,
+                Snackbar.LENGTH_LONG
```

Comment:
Nitpick: I found the duration of the snack bar notification a bit short to read the error message. However, I noticed we use the widely parameter `LENGTH_LONG` across the Android app code base, so maybe it's a standard time to wait.